lib/node_modules/@stdlib/utils/parse-ndjson/package.json

```diff
@@ -0,0 +1,68 @@
+{
+    "name": "@stdlib/utils/parse-ndjson",
```

Comment:
The indentation in this file is off. It should be two-space indentation.

Reply:
Resolved